tests/osgi-tests/src/test/java/org/wso2/is/portal/user/client/api/unit/test/RecoveryMgtServiceTest.java

```diff
@@ -81,8 +82,15 @@ public void testGetRecoveryConfig() throws UserPortalUIException {
 
     @Test(groups = "getRecoveryQuestions", dependsOnGroups = {"getRecoveryConfig"})
     public void getUserChallengeQuestionAtOnce() throws UserPortalUIException, IdentityRecoveryException {
+        Map<String, String> userClaims = new HashMap<>();
+        Map<String, String> credentials = new HashMap<>();
+        userClaims.put(UserPotalOSGiTestConstants.ClaimURIs.USERNAME_CLAIM_URI, "Ayesha");
+        userClaims.put(UserPotalOSGiTestConstants.ClaimURIs.FIRST_NAME_CLAIM_URI, "Ayesha");
+        userClaims.put(UserPotalOSGiTestConstants.ClaimURIs.LAST_NAME_CLAIM_URI, "Dissanayaka");
+        userClaims.put(UserPotalOSGiTestConstants.ClaimURIs.EMAIL_CLAIM_URI, "ayesha@wso2.com");
 
-        addUser();
+        credentials.put("password", "password");
```

Comment:
use UserPotalOSGiTestConstants.UserPotalOSGiTestConstants.PASSWORD_CALLBACK as the key